src/main/java/com/unityTest/testrunner/service/CodeService.java

```diff
@@ -171,8 +174,33 @@ private void writeTestCaseToSuiteFile(PLanguage lang, File testFile, SuiteFile s
 		// Write suite file content
 		bos.write(suiteFile.getContent());
 		// Write test case content to file depending on language
-		// TODO FIX THIS
-		bos.write(String.format("\n%s", Utils.indent(caze.getCode(), 1)).getBytes());
+		switch (lang) {
+			case JAVA:
+				// TODO;
+				break;
+			case HASKELL:
+				StringBuilder haskellBuilder = new StringBuilder();
+
+				// Importing Hunit
+				haskellBuilder.append("import Test.HUnit\n\n");

```

Comment:
This should prepend a `\n` incase the suite file doesn't end on an empty line.

Reply:
Something like this?
```suggestion
				haskellBuilder.append("\nimport Test.HUnit\n\n");

```

Reply:
Pretty much 😆